setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import find_packages, setup
+
+import yolapy
+
+
+setup(
+    name='yolapy',
+    version=yolapy.__version__,
```

Comment:
The root __init__.py file doesn't import anything. Here's why click needs to do what you're suggesting: https://github.com/mitsuhiko/click/blob/master/click/__init__.py

I wouldn't recommend using the approach you've outlined here until such a time that it becomes explicitly necessary. If we're worried about someone adding imports to __init__.py and not realizing that they are breaking setup.py by doing so, then let's just add a comment to __init__.py that says that setup.py depends on it and that setup.py will need to be changed if imports are added.